src/net.cpp

```diff
@@ -99,6 +99,7 @@ map<CInv, CDataStream> mapRelay;
 deque<pair<int64_t, CInv> > vRelayExpiration;
 CCriticalSection cs_mapRelay;
 limitedmap<CInv, int64_t> mapAlreadyAskedFor(MAX_INV_SZ);
+limitedmap<CInv, int64_t> mapAlreadyReceived(MAPRECEIVED_MAX_SZ);
```

Comment:
Is it ok those two limitedmap having different size?

Reply:
It should be ok, the two maps have different purposes and are independent of each other, so it is reasonable they have different sizes.